content/blog/sprint-review-2003.md

```diff
@@ -0,0 +1,45 @@
+Title: Wazo Platform 20.03 Released
+Date: 2020-02-24
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2003
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 20.03 release.
+
+## New features in this sprint
+
+* **Users**: Import and export of users now include the subscription field. This field is not used directly by Wazo Platform, but it can be used by external applications to categorize users and give them more or less permissions depending on their subscription value.
```

Comment:
IMO we should not communicate about this field
It's a false idea that someone can use external application to categorize user, because the day we will need to migrate values, we will do it without thinking to other use cases...